packages/cdk/apps/core/app.ts

```diff
@@ -37,7 +37,7 @@ new CoreStack(app, `${PRODUCT_NAME}-Core`, {
     },
     {
       name: "installed-artifacts/s3-root-url",
-      value: `s3://${bucketName}/batch-artifacts`,
+      value: `s3://${bucketName}/artifacts/batch-artifacts`,
```

Comment:
How about make this "_common" or "_agc" in case we want to put other things under this prefix?